public/static/covizu/js/customOptions.js

```diff
@@ -1,15 +1,55 @@
 // get data info from current URL
 var urlParams = new URLSearchParams(window.location.search);
-var dataVersion = urlParams.get('dataVersion') || '';
+// our covizu version number. may be different from upstream's version number.
+var covizuVersion = '1.0.0';
 var dataUrl = urlParams.get('dataUrl') || '';
+var filesUrl = urlParams.get('filesUrl') || '';
 
+// get a list of clusters files from the current data version folder
+// in order to get the date of the latest file upload
+var clusters = [];
+var filesPath = window.location.hostname.includes('localhost')
+  ? 'cluster-list.json'
+  : '?format=json&prefix=' + covizuVersion + '/clusters.20';
```

Comment:
checked the search API docs, this is the best way to query version number folders.